Sources/NIOWebSocket/WebSocketUpgrader.swift

```diff
@@ -0,0 +1,126 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+//
```

Comment:
delete Xcode file header

Reply:
Why are we deleting the XCode file header? We don't do it anywhere else.

Reply:
we don't have any of the default Xcode file headers anywhere really (except two which I fixed in #139 ).